.github/workflows/end2end.yaml

```
@@ -445,6 +447,16 @@ jobs:
       - name: Debug wait
         uses: ./.github/actions/debug-wait
         timeout-minutes: 60
+      - name: Upload results
+        if: "!cancelled() && env.TRUNK_TOKEN"
```

should we `[continue-on-error](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error)`, to avoid any unexpected failures ?

```suggestion
        if: "!cancelled() && env.TRUNK_TOKEN"
        continue-on-error: true
```

Then ideally we need to have something to track when this is broken... but maybe it is safer to keep it possibly failing for now, as it is not even in beta yet?

---

I think the point is you just don't want your pipeline to fail if trunk happens to be facing a brief service interruption.

If you want to track when it's broken, you can [track each step failure rate](https://app.trunk.io/scality/scality/Zenko/ci-analytics/1_01db756e-9927-4cfb-b98d-dc56e2952ed2/1_dc061645-8e2f-4d67-9087-7630167d9ee3) with trunk

---

that is nice! but it cannot track failures of trunk upload step... (inception)